### ISSUE TITLE
Mint Law Issue

### ISSUE THREAD

**Issue by techop2000** [StrangeLoopGames/EcoIssues#8711]
We are on a dedicated server. We have learned the skill, created, and placed 2 mints. We have also minted currency on the mint. We cannot however see the mint listed in the propose law area: On Player Action when attempting to craft where object is (.......). As far as I can tell, we can see any other crafting table in the list.

**Comment by MarjieAVolk** [comment]
Ah, yeah you need to look under 'where item is ...', not 'where object is...'.  The naming is rather confusing.  In this case, the 'item' is the item you are crafting, and the 'object' is the crafting station you are crafting at.  So you would use for example 'object is blast furnace' to prevent people from using a blast furnace, but you would use 'item is blast furnace' to prevent people from crafting a blast furnace.

**Comment by techop2000** [comment]
you are correct about what you said but that is the exact problem we have. we are able to prevent people from crafting new mints, but we are not able to limit their use of the mint. We want to make a law that says users can only use the mint to create currency a set amount of times per day that passes.

**Comment by techop2000** [comment]
in other words, it does show up in where item is, but does not show up in where object is, and it should.

**Comment by techop2000** [comment]
As a workaround right now, we had to back the currency with torches then pass a law limiting the amount of torches a person can produce per day. another way it could be improved would be to allow a law param of craft where object is (.......)       and (.....) would be a list of all crafted currencies.

**Comment by MarjieAVolk** [comment]
Okay, we've already got a request to add that, here: https://github.com/StrangeLoopGames/EcoIssues/issues/5428

I was leaving this issue open to remind myself to think about more clear text for 'item is' vs 'object is', but maybe that was not actually an issue for you.

**Comment by techop2000** [comment]
no, the difference between item and action was understood by me. I already have a law structure built that would limit the production of currency at mints, assuming i understand the parameters correctly. It just doesn't work currently because mints aren't included in the where object is param. Here is the law;

on player action when attempting to craft where object is (bloomery)
condition
if numeric comparison actions your craft actions where object is (bloomery) < Calculated value current time (days) * constant (2)    
  then allow
otherwise Prevent

substituting (mint) for (bloomery) if "object is (Mint)" existed, then this should count the number of times someone used mints to craft currency and prevent it if the value exceeded 2x the current time of the server. 

In regard to the item vs object problem. For me "item is was very clear". "object is" was a little confusing until i looked at the list after it, then it was also very clear. A solution would be changing "object is" to something like "machine is" or "Crafting table is". You just need to figure out what word best sums up all of your crafting objects.

p.s. Thanks for the quick responses, and I am really enjoying Eco!
